docs/install/usb-install.md

```diff
@@ -30,6 +30,14 @@ Make sure you choose the correct device. The process erases data on the selected
 
 ## Common issues
 
+###
+
+Sometimes using a USB to mount the Harvester ISO disk directs you to Grub2. In this case, it helps to know how to set the root partition and where to locate `grub.cfg`.
+
+You can see the current root partition in the `grub.cfg`. To view the `grub.cfg` from the ISO, run the command `chainloader /boot/grub2/x86_64-efi/grub.efi`.
+
+To set a new root partition, run the command `set root=(hdN,msdos1)`, where N denotes N numbers of possible drives. N should be the drive your USB is mounted to.
+
```

Comment:
@irishgordo In [issue#3590](https://github.com/harvester/harvester/issues/3590), you mentioned "Then they'd be able to fire off the boot command from Grub2 and be taken to the interactive ISO installation." Should that step be added here as well?

Reply:
Ahh I see what you're saying :smile:  - I've added a suggestion below :+1: